crates/iota/src/keytool.rs

```diff
@@ -1163,7 +1163,7 @@ impl From<PublicKey> for Key {
         Key {
             alias: None, // this is retrieved later
             iota_address: IotaAddress::from(&pk),
-            public_base64_key: pk.encode_base64(),
+            public_base64_key: Base64::encode(pk.as_ref()),
```

Comment:
Wouldn't that affect more places than just `keytool show`?

Reply:
Yes, `ExportedKey` contains it and `CommandOutput::{Generate, Import, List, Show}`, but I think it makes sense for all of them to change